setup.py

```diff
@@ -6,56 +6,40 @@
 This package uses 3rd party submodules that might be published under other comparable open licenses.
 
 This module is a grapheme-to-phoneme (g2p) program for Icelandic.
+The _post_install() only gets called when installing from repository via setup.py install.
+If installing from pip, it is necessary to run `fetch_models` before running g2p.
+
```

Comment:
what happens, if the models aren't existing, because the user missed to call `fetch_models` ? I.e. is then a message printed, how he/she can download the model files ?